README.md

```diff
@@ -64,8 +64,12 @@ Fern is a Golang Gin-based API that connects to a PostgreSQL database. It is des
 
 ### Accessing Test Reports
 
-- View reports at `http://[your-api-url]/reports/testruns`.
-- If using `make docker-run`, reports are available at `http://localhost:8080/reports/testruns`.
+To view the test reports, you can use the Fern-UI frontend,
+which provides a visual dashboard for the test results.
```

Comment:
Lets leave both options ie. use fern-ui or use the embedded UI. The idea behind this is to foster innovation outside if needed using the apis we have